packages/angular-teleport/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { TeleportDirective } from './lib/teleport.directive';
```

Comment:
You also need to export a Module

Reply:
Nope. With Angular 14 there is the `standalone` option.